package.json

```diff
@@ -31,7 +27,8 @@
     "sinon": "^9.2.2"
   },
   "engines": {
-    "node": ">=12.0.0"
+    "node": ">=12.0.0",
```

Comment:
Should we at least bump this to node@14 since 12 is no longer LTS?

Reply:
I do not think so. Node 12 works. People still using Node 12 would get forced to update if we set this to 14.

Reply:
While true. It is no longer receiving updates. I don't think we should be expected to support folks using node 12 for this CLI.